Resources/Prototypes/Entities/Mobs/Species/harpy.yml

```
@@ -118,10 +118,10 @@
   - type: LanguageKnowledge
     speaks:
     - TauCetiBasic
-    - Tradeband
+    - ValyrianStandard
     understands:
     - TauCetiBasic
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)